### DISCUSSION TITLE
Get the version of extension in the extension

### DISCUSSION THREAD

**Discussion by rokdd** [directus/directus#14883] [Archive]
Not so much to say more.. How I can manage to display the extension version inside of the extension? Theoretically there is package.json but in the dist this is not relevant anymore.. or should it be "only" hardcoded as a variable?

**Comment by br41nslug** [comment]
In the final build version it has to be "hardcoded" into the `index.js` but you could load that from the package.json at build time.

**Reply by rokdd** [reply]
What does it help me at build time? Just to understand how you mean it.. By the way thanks for your great help always ;-)

**Reply by br41nslug** [reply]
I intended that as an alternative to hardcoding it in the source of your extension like `const VERSION='1.2.3';`.
By having something like 
```
import pkg from "../package.json";
const VERSION = pkg.version;
``` 
(if that works 😬 i think it should)
That would be resolved during the build of your extension essentially hardcoding it into the final extension but without having to keep 2 places with versions in sync

**Reply by rokdd** [reply]
Great it worked!
